mdx-layout-tailwind/src/styles.css

```diff
@@ -0,0 +1,19 @@
+article {
```

Comment:
I think tailwind should not be parametrized like that but rather with classes inside the layout. I think we should style them with components provided by the MDXProvider https://mdxjs.com/getting-started#working-with-components ... like
 ```
const components = {
li : ...
ul : ...
}

export const Layout = ()=> (<MDXProvider components={componnents}>....</MDXProvider>)

```

Then I think it should be styled correctly and follow the Tailwind-style